### PR TITLE
Relay waitlist

### PR DESCRIPTION
**Pull request by nathan-barrett**
## Summary

Adds relay waitlist pages to bedrock for the VPN + Relay Bundle and Phone Masking
- Phone masking wait list page https://relay.firefox.com/phone/waitlist/
- VPN / Relay bundle wait list page https://relay.firefox.com/vpn-relay/waitlist/

Something to note - the original relay pages use notification bars, but I decided to use the standard newsletter Thank you messaging instead of notification bars

## Issue / Bugzilla link
#12933

## Testing

To test this work:
- http://localhost:8000/products/relay/waitlist/vpn/
- http://localhost:8000/products/relay/waitlist/phone/

### CODE REVIEW

File: bedrock/products/templates/products/relay/waitlist/phone.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "products/relay/includes/waitlist-base.html" %}
+
+{% set product = ftl('waitlist-phone-name') %}
+
+{% block waitlist_heading %}{{ ftl('waitlist-heading-phone') }}{% endblock %}
+{% block waitlist_lede %}{{ ftl('waitlist-lead-phone') }}{% endblock %}
```
Comment (stephaniehobson) [inline bedrock/products/templates/products/relay/waitlist/phone.html:12]: Standardize spelling of lede/lead please.

File: media/css/products/relay/waitlist.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    }
+}
+
+.c-waitlist-success {
+    text-align: center;
+}
+
+.mzp-c-form {
+    i {
+        display: block;
```
Comment (stephaniehobson) [inline media/css/products/relay/waitlist.scss:27]: You can use [fieldnotes](https://protocol.mozilla.org/components/detail/field-notes.html) for this too.

**Review by stephaniehobson — COMMENTED** [review]
It's getting late so I didn't do a full review but I'll post what I noticed so you don't have to wait on me to start changes.

- Both pages are missing a page title and favicon.
- I think it's worth creating a new string for the phone masking success message.

It's looking good! FWIW I like the standard newsletter thank you messaging better than the notification bar. 💅

File: bedrock/products/templates/products/relay/waitlist/phone.html (hunk 1 above)
Comment (nathan-barrett) [reply bedrock/products/templates/products/relay/waitlist/phone.html:12]: Oh we can, I just spelled the block name differently :)

**Pull request merged**